src/test/java/com/epam/training/array/ArrayLogicTest.java

```diff
@@ -1,7 +1,151 @@
 package com.epam.training.array;
 
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+
 public class ArrayLogicTest {
 
-    //TODO: add tests for various scenarios
+	@Test
+	public void testExistsElement(){
```

Comment:
naming for test should be like for example testExistsShouldDoSomsingWhenDataIsValid